src/Shaders/ShadersInclude/morphTargetsVertex.fx

```diff
@@ -3,5 +3,10 @@
 	
 	#ifdef MORPHTARGETS_NORMAL
 	normalUpdated += (normal{X} - normal) * morphTargetInfluences[{X}];
-	#endif	
+	#endif
+
+	#ifdef MORPHTARGETS_TANGENT
+	tangentTemp = tangentUpdated.xyz + ((tangent{X} - tangent.xyz) * morphTargetInfluences[{X}]);

```

Comment:
Tangent from morph targets is a vec3 delta on xyz, not on w. w should stay -1/+1. Is there a better way to do that?